docs/products/redis/concepts/high-availability-redis.rst

```diff
@@ -5,44 +5,62 @@ Aiven for Redis®* is available on a variety of plans, offering different levels
+- **Severe failure**: In case of severe hardware or software problems, such as losing an entire node, it requires more drastic recovery measures. An entire node (virtual machine) may be lost due to hardware failure or a severe software issue. Aiven's monitoring infrastructure automatically detects a failing node when it reports problems with its self-diagnostics or stops communicating altogether. The monitoring infrastructure then schedules the creation of a new replacement node.
+
 
 .. Note::
-        In case of database failover, the **Service URI** of your service remains the same, only the IP address changes to point to the new master node.
+        In case of database failover, your service's **Service URI** remains the same—only the IP address changes to point to the new master node.
+
 
 Highly available business, premium and custom service plans
 ------------------------------------------------------------
 
-When the failed node is a Redis **standby**, the **primary** node keeps on running normally, and provides normal service level to the client applications. Once the new replacement standby node is ready and synchronized with the master, it starts replicating the master in real time as the situation reverts back to normal.
+If a Redis node fails and is a **standby**, node, the **primary** node continues to run normally, providing uninterrupted service to client applications. Once a new replacement standby node is ready and synchronized with the main, it replicates it in real time until the situation returns to normal.
 
-When the failed node is a Redis **primary**, the combined information from the Aiven monitoring infrastructure and the standby node is used to make a failover decision. The standby node is then promoted as the new primary and immediately starts serving clients. A new replacement node is automatically scheduled and becomes the new standby node.
+When the failed node is a Redis **primary**, the combined information from the Aiven monitoring infrastructure and the standby node is used to make a failover decision. The standby node is promoted as the new primary and immediately serves clients. A new replacement node is automatically scheduled and becomes the new standby node.
 
-If all the **primary** and **standby nodes** fail at the same time, new nodes are automatically scheduled for creation to become the new primary and standby. The primary node is restored from the latest available backup, which could involve some degree of data loss. Namely all the writes to the database since the last backup will be lost.
+If both the **primary** and **standby** nodes fail simultaneously, new nodes will be created automatically to take their place as the new primary and standby. However, this process will involve some degree of data loss since the primary node will be restored from the most recent backup available. Therefore, any writes made to the database since the last backup will be lost.
```

Comment:
```suggestion
If both the **primary** and **standby** nodes fail simultaneously, new nodes will be created automatically to take their place as the new primary and standby. However, this process involves some degree of data loss since the primary node is restored from the most recent backup available. Therefore, any writes made to the database since the last backup will be lost.
```

Reply:
Made a suggestion to just change a couple of these to use present tense to fit with style guide since it seems appropriate in those two particular cases (but would not be in all cases in this paragraph).